### PR TITLE
Removed Abicase from Dev Pane and added missing update note

### PR DESCRIPTION
**Pull request by jani270**

### CODE REVIEW

**Review by romangraef — DISMISSED** [review]

File: Update Notes/2.1.1.md [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
- - Remove custom option toggle textures in /neu - nopo
- - Fix Pickaxe Cooldown not being correct on first join - Ascynx
- - Fix social exp not being correct in /pv - Vixid
- - Fix a crash when downgrading from future NEU versions - nopo
+ - Fixed /join command not working for uppercase floors - walker or whalker
+ - Removed custom option toggle textures in /neu - nopo
+ - Fixed Pickaxe Cooldown not being correct on first join - Ascynx
+ - Fixed social exp not being correct in /pv - Vixid
+ - Fixed a crash when downgrading from future NEU versions - nopo
  - Unify writing conventions for "SkyBlock" and similar - walker or whalker
```
Comment (romangraef) [inline Update Notes/2.1.1.md:18]: bruh

**Review by romangraef — DISMISSED** [review]
anyway this is wrong and stupid and wrong and im sad and you should feel bad

**Review by hannibal002 — DISMISSED** [review]

**Comment by jani270** [comment]
> anyway this is wrong and stupid and wrong and im sad and you should feel bad

I will consider not feeling bad

**Comment by romangraef** [comment]
image
At least feel this

**Review by romangraef — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**